src/runtime/cmd/kata-monitor/main.go

```diff
@@ -119,6 +172,14 @@ func main() {
 	logrus.Fatal(svr.ListenAndServe())
 }
 
+func indexPage(w http.ResponseWriter, r *http.Request) {
+	w.Write([]byte("Available HTTP endpoints:\n"))
+
+	for i := range endpoints {
+		w.Write([]byte(fmt.Sprintf("%-25s: %s\n", endpoints[i].path, endpoints[i].desc)))
```

Comment:
The parameter `%-25s` depends on the actual output of the index page(the `path` property's length).
And I suggest adding a comment that when a new endpoint is added,  the index page should be checked that whether the output has corrupted.

Reply:
Right! Well, just changed to retrieve the max length dynamically now (🤔  too much for something we will not update that frequently? well, the overhead would be ridiculous btw).
Thanks @liubin  :-)